dbdimp.h

```diff
@@ -49,6 +49,7 @@
 #define NEW_DATATYPE_VERSION 50003
 #define SSL_VERIFY_VERSION 50023
 #define SSL_LAST_VERIFY_VERSION 50799
+#define SSL_MODE_VERSION 50711
```

Comment:
Is this version correct for both mysql and mariadb?

@mbeijen looks like we need some travis tests for both mysql and mariadb ideally for more versions...